app/src/main/java/org/nv95/openmanga/providers/staff/Providers.java

```diff
@@ -38,7 +39,8 @@
             new ProviderSummary(10, "Яой-тян", YaoiChanProvider.class, Languages.RU, R.xml.pref_anychan),
             new ProviderSummary(11, "TruyenTranh", TruyenTranhProvider.class, Languages.VIE, 0),
             new ProviderSummary(12, "Хентай-тян", HentaichanProvider.class, Languages.RU, R.xml.pref_henchan),
-            new ProviderSummary(13, "ScanFR", ScanFRProvider.class, Languages.FR, 0)
+            new ProviderSummary(13, "ScanFR", ScanFRProvider.class, Languages.FR, 0),
+            new ProviderSummary(14, "HeavenManga", HeavenMangaProvider.class, Languages.ES, 0)
```

Comment:
the app has no translation for es, please add translate in values-es or change to en

Reply:
This is not related to the application language, so strings translation is not required (but desirable)